OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
```

Comment:
Will it mean that we'll be pinged to all PRs in the repo even the tasks we don't have relation to?